docs/GettingStartedDocs/Windows_vscode.md

```diff
@@ -0,0 +1,163 @@
+
+![Run](images/VSCodeRun.png)
+
+## Configuring Intellisense
+
+Intellisense should work out of the box for files within your workspace. However, Intellisense may not be aware of where to locate the Open Enclave SDK headers.
+Open settings.json under the .vscode folder and add entries for "C_Cpp.default.includePath" and "C_Cpp.default.systemIncludePath".
+If you installed the Open Enclave SDK at `C:\openenclave`, the Settings.kson would be as below:
+
+```json
+{
+    "cmake.configureArgs": [
+        "-DNUGET_PACKAGE_PATH=C:\\openenclave_prereqs\\prereqs\\nuget",
+        "-DOpenEnclave_DIR=C:\\openenclave\\lib\\openenclave\\cmake"
+    ],
+    "C_Cpp.default.includePath": ["C:\\openenclave\\include"],
+    "C_Cpp.default.systemIncludePath": [
+        "C:\\openenclave\\include\\openenclave\\3rdparty\\libc",
+        "C:\\openenclave\\include\\openenclave\\3rdparty\\libcxx"
+    ]
+}
+```
+
+## Debug Your Open Enclave Application
+
+Add a CDB Debug Configuration as shown below.
+
+![CDB Debug Configuration](images/VSCodeDebugConfiguration.png)
+
+Fill in program path, parameters and other values in the configuration.
+
+![Edit CDB Debug Configuration](images/VSCodeEditDebugConfiguration.png)
+
+Here is an example of launch.json after editing it.
+
+```json
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Launch",
+            "type": "cdb",
+            "request": "launch",
+            "program": "${workspaceRoot}/build/host/helloworld_host.exe",
+            "sourcepath": "${workspaceRoot};C:/ignite/openenclave-sdk-src",
+            "workingdirectory": "${workspaceRoot}/build",
+            "debugthedebugger": false,
+            "initialbreak": false,
+            "initialcommands": ".sympath \"\"",
+            "args": "enclave/enclave.signed"
+        }
+    ]
+}
+```
+
+Open host.c and add a breakpoint. Start debugging.
+
+![Host Breakpoint](images/VSCodeHostBreakpoint.png)
+
+Step over the line that creates the enclave. The Console pane should show that the enclave has been loaded.
+
+![Stop After Enclave Creation](images/VSCodeStopAfterEnclaveCreation.png)
+
+Note: [CDB commands](https://docs.microsoft.com/en-us/windows-hardware/drivers/debugger/) can be executed in the Console Prompt.
+
+![Debug Console](images/VSCodeDebugConsole.png)
+
+Open enc.c and put a breakpoint and continue execution.
+
+![Enclave Breakpoint](images/VSCodeEnclaveBreakpoint.png)
+
+
+## Known Issues
+
+The VS Code Debugger extension is currently released as an early preview.
+These issues are being worked on and will be fixed in an upcoming update.
+
+- Breakpoints aren't yet completely persisted and restored correctly.
```

Comment:
Grammar: missing commas after "therefore". Also, missing article before "debugging session".

> - Breakpoints aren't yet completely persisted and restored correctly.
Therefore, it is recommended that you clear all breakpoints and set them again every time the program is launched for debugging.
> - _The_ debugging session is not terminated when the program runs to completion.
Therefore, it is recommended that you stop and start a new debugging session each time.

Reply:
fixed